news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+
+Playing with a tablet is a completely different ball game compared to the mouse, but rather than being a direct upgrade by eliminating mouse drift with its absolute positioning, it brings up new challenges and questions regarding its ideal usage. Ideal pen grip, tablet area, dragging or hovering play styles have all been detrimental to increasing the learning curve, which is why some players choose to remain on the much more familiar and intuitive mouse.
+
+Over the years many top players and regular players started switching over from mouse to tablet in order to avoid issues associated with mice, however there have always been top players with both input devices, and they are both considered viable for top-level play, each with its own upsides and downsides.
+
+A great video by community member [Willy](https://osu.ppy.sh/users/3521482) outlines what's written here and more regarding the usage of tablets as input devices:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/V3K3C94fs1o" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Gaming mechanical keyboards (2010)
+
+Mechanical keyboards have been a thing for decades, however after Razer put [gaming and mechanical switches into a compelling package](https://press.razer.com/product-news/the-worlds-first-mechanical-keyboard-designed-for-gaming-launched-at-gamescom-2010/), that launched the chain reaction that resulted in the mechanical keyboard market we know of today.
+
+By 2010 osu! players were already shifting away from clicking circles with their mouse, to clicking with their Z and X keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups, players would start using them for osu! as well. And while there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is suggesting to newer players to purchase a mechanical keyboard, as it provides better feedback when a key is pressed.
+
+The mass adoption of mechanical keyboards did not only occur in the osu! community, but in the gaming community as a whole, and you'd be hard-pressed to find a gaming setup nowadays without one of these smack-dab in the middle of any gamer's desk.
+
+## High refresh rate LCD monitors (2012)
+
+Around mid 2012 first LCD high refresh rate monitors were released with the rates as high as 144hz. Even before LCD monitors were a thing CRT's had the capability of refreshing up to 100hz in most cases (and even higher for high-end models), however with the introduction of high refresh rate LCD panels this technology slowly became much easier to adopt, and is now a staple of any modern gaming setup.
```

Comment:
```suggestion
Around mid 2012, the first LCD high-refresh rate monitors were released with refresh rates of up to 144 Hz. Even before LCD monitors were a thing, CRT monitors had the capability of refreshing up to 100 Hz in most cases (and even higher for high-end models). However with the introduction of high-refresh rate LCD panels this technology slowly became much easier to adopt, and is now a staple of any modern gaming setup.
```

Remove duplicate use of "high" in first sentence.
[NIT] Hertz should be styled as "100 Hz" (note space and capitalization)